docs/Networking.asciidoc

```diff
@@ -48,235 +48,275 @@ automatically computed from $worker_id if not set.
 In TAP mode the system administrator is expected to configure the
 network, required internet access etc on the host manually.
 
-TAP devices need be owned by the +_openqa-worker+ user for openQA to
-be able to access them.
+=== VDE Based Network
 
-[source,sh]
----------------
-tunctl -u _openqa-worker -p -t tap0
----------------
+Virtual Distributed Ethernet provides a software switch that runs in
+user space. It allows to connect several qemu instances without
+affecting the system's network configuration.
 
-If you want to use TAP device which doesn't exist on the system,
-you need to set CAP_NET_ADMIN capability on qemu binary file:
+The openQA workers need a vde_switch instance running. The workers
+reconfigure the switch as needed by the job.
 
-[source,sh]
----------------
-zypper in libcap-progs
-setcap CAP_NET_ADMIN=ep /usr/bin/qemu-system-x86_64
----------------
+==== Basic, single machine tests
+
+To start with a basic configuration like qemu user mode networking,
+create a machine with the following settings:
+
+- +VDE_SOCKETDIR=/run/openqa+
+- +NICTYPE=vde+
+- +NICVLAN=0+
 
-Network setup can be changed after qemu is started using network configure script
-specified in TAPSCRIPT variable.
+Start switch and user mode networking:
 
-Sample script to add TAP device to existing bridge br0:
 [source,sh]
 ---------------
-sudo brctl addif br0 $1
-sudo ip link set $1 up
+systemctl start openqa-vde_switch
+systemctl start openqa-slirpvde
 ---------------
 
-== TAP with Open vSwitch
+With this setting all jobs on the same host would be in the same
+network share the same SLIRP instance though.
```

Comment:
Honestly, I did not change that part from the legacy documentation. I just moved it to another position in the documentation. As my task was to provide documentation/step-by-step guide for setup multimachine network, I did not verify the internal structure and how the Open VSwitch works. I've kept it as is. So, it might be discussed in another topic, I guess.